### PR TITLE
Updates the Terraform auto-plan workflow

### PR DESCRIPTION
**Pull request by mike-schenk**
- adds -input=false to the terraform commands to avoid silent stalls in actions runs

### CODE REVIEW

**Review by izep — APPROVED** [review]
I didn't know about that parameter...

File: workflow-templates/im-build-tf-auto-plan-and-comment-on-prs.yml [hunk 1]
```diff
@@ -1,126 +1,126 @@
...
-             
-            __Pusher__: @${{ github.actor }}
-            __Action__: \`${{ github.event_name }}\``;
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: output
-            })
+# Workflow Code: DeterminedPorcupine_v23    DO NOT REMOVE
```
Comment (JosephDSchwartz) [inline workflow-templates/im-build-tf-auto-plan-and-comment-on-prs.yml:1]:
```suggestion
# Workflow Code: DeterminedPorcupine_v24    DO NOT REMOVE
```

**Review by JosephDSchwartz — APPROVED** [review]

**Comment by danielle-casella-adams** [comment]
> I didn't know about that parameter...

Same.  

[-input=true](https://developer.hashicorp.com/terraform/cli/commands/init#input-true) Ask for input if necessary. If false, will error if input was required.

I love this addition @mike-schenk!

**Comment by mike-schenk** [comment]
> > I didn't know about that parameter...
> 
> Same.
> 
> [-input=true](https://developer.hashicorp.com/terraform/cli/commands/init#input-true) Ask for input if necessary. If false, will error if input was required.
> 
> I love this addition @mike-schenk!

The reason I added it here is because I had a situation where a root module required a variable that wasn't provided, so Terraform was asking for the input... but in GitHub actions, that simply appears as a stalled step that eventually just fails with a timeout and with nothing in the logs for that step.

**Pull request merged**